compiler/quilt/tools/main.py

```diff
@@ -13,18 +13,23 @@
 from . import command
 from .const import DEFAULT_QUILT_YML
 
+# Importing this monkey-patches `command` if testing the CLI in subprocess mode
+from ..test import test_cli     # pylint:disable=W0611
```

Comment:
Importing this in production code feels wrong... I think doing a conditional import and patching would be better. Also, make monkey-patching a helper function instead of a side-effect of import. So something like this:

```
if os.environ.get('QUILT_TEST_CLI_SUBPROC') == "True":
  from ..test import test_cli
  test_cli.monkey_patch(...)
```

Reply:
Yeah, I originally did something similar.  I'll go back to what I had.  I was mainly just trying to get the explicitly testing-related code out of main.py.